<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="4" skipped="14" tests="50" time="5.137" timestamp="2025-03-08T13:11:25.000747+05:30" hostname="Amans-MacBook-Pro.local"><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_greater[9-5]" time="0.001" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_greater[5-9]" time="0.000"><failure message="AssertionError: 5 is not greater than 9&#10;assert 5 &gt; 9">self = &lt;test_module02.TestCases object at 0x105c29f10&gt;, a = 5, b = 9

    @pytest.mark.parametrize("a, b", testset)
    def test_greater(self,a,b):
&gt;       assert a &gt; b, f"{a} is not greater than {b}"
E       AssertionError: 5 is not greater than 9
E       assert 5 &gt; 9

pytest-topics/pytest-assertions/test_module02.py:10: AssertionError</failure></testcase><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_greater[5-4]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_greater[10-5]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_true" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_cmpr_string_char" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_divmod[9-5]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_divmod[5-9]" time="0.000"><failure message="AssertionError: 1 is not in divmod(5,9)&#10;assert 1 in (0, 5)&#10; +  where (0, 5) = divmod(5, 9)">self = &lt;test_module02.TestCases object at 0x105c3d190&gt;, a = 5, b = 9

    @pytest.mark.parametrize("a, b", testset)
    def test_divmod(self,a,b):
&gt;       assert 1 in divmod(a,b), f"1 is not in divmod({a},{b})"
E       AssertionError: 1 is not in divmod(5,9)
E       assert 1 in (0, 5)
E        +  where (0, 5) = divmod(5, 9)

pytest-topics/pytest-assertions/test_module02.py:21: AssertionError</failure></testcase><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_divmod[5-4]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_divmod[10-5]" time="0.000"><failure message="AssertionError: 1 is not in divmod(10,5)&#10;assert 1 in (2, 0)&#10; +  where (2, 0) = divmod(10, 5)">self = &lt;test_module02.TestCases object at 0x105c3d690&gt;, a = 10, b = 5

    @pytest.mark.parametrize("a, b", testset)
    def test_divmod(self,a,b):
&gt;       assert 1 in divmod(a,b), f"1 is not in divmod({a},{b})"
E       AssertionError: 1 is not in divmod(10,5)
E       assert 1 in (2, 0)
E        +  where (2, 0) = divmod(10, 5)

pytest-topics/pytest-assertions/test_module02.py:21: AssertionError</failure></testcase><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_find_in_string" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module02.TestCases" name="test_is_not_in_string" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_zero_divisibility" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_equation[1-2]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_equation[2-1]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_404" time="1.212" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_error_assert" time="0.001" /><testcase classname="pytest-topics.pytest-assertions.test_module03.TestCases" name="test_tuple_cmpr" time="0.001" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_conversion[8-46.4]" time="0.001" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_conversion[42-107.6]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_conversion[100-212.0]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_conversion[23-73.4]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_conversion[35-95.0]" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_no_input" time="0.000" /><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_datatype_confirm_float[8]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:30: Skipping Test</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_datatype_confirm_float[42]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:30: Skipping Test</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_datatype_confirm_float[100]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:30: Skipping Test</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_datatype_confirm_float[23]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:30: Skipping Test</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_datatype_confirm_float[35]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:30: Skipping Test</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_404" time="0.000"><skipped type="pytest.skip" message="Don't execute this for python version above 3.8">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/pytest-assertions/test_module04.py:37: Don't execute this for python version above 3.8</skipped></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_404_xfail" time="1.218"><skipped type="pytest.xfail" message="Don't execute this for python version above 3.8" /></testcase><testcase classname="pytest-topics.pytest-assertions.test_module04.TestCases" name="test_no_input_xpass" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_conversion[8-46.4]" time="0.001" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_conversion[42-107.6]" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_conversion[100-212.0]" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_conversion[23-73.4]" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_conversion[35-95.0]" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_no_input" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_datatype_confirm_float[8]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:33: Skipping Test</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_datatype_confirm_float[42]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:33: Skipping Test</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_datatype_confirm_float[100]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:33: Skipping Test</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_datatype_confirm_float[23]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:33: Skipping Test</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_datatype_confirm_float[35]" time="0.000"><skipped type="pytest.skip" message="Skipping Test">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:33: Skipping Test</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_404" time="0.001"><skipped type="pytest.skip" message="Don't execute this for python version above 3.8">/Users/akd/Github/pytest-tutorial/pytest-automation/pythonProject/pytest-topics/test_marker_skip.py:40: Don't execute this for python version above 3.8</skipped></testcase><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_str_slice" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_str_split" time="0.000" /><testcase classname="pytest-topics.test_marker_skip.TestCases" name="test_404_xfail" time="2.439"><skipped type="pytest.xfail" message="Expected to fail" /></testcase><testcase classname="pytest-topics.test_module01" name="test_addition" time="0.000" /><testcase classname="pytest-topics.test_module01" name="test_subtraction" time="0.000"><failure message="AssertionError: Intentional failure 1&#10;assert (5 - 5) != 0">def test_subtraction():
&gt;       assert 5 - 5 != 0, "Intentional failure 1"
E       AssertionError: Intentional failure 1
E       assert (5 - 5) != 0

pytest-topics/test_module01.py:6: AssertionError</failure></testcase><testcase classname="pytest-topics.test_module01" name="test_integer_division" time="0.000" /></testsuite></testsuites>